app/src/autocomplete.js

```diff
@@ -76,6 +76,7 @@ class Autocomplete {
       const sizeModifier = this._sizeModifier(dropdownMenuWidth);
       const nbSnippetWords = this._nbSnippetWords(dropdownMenuWidth);
       const params = {
+        analytics,
```

Comment:
Same here, but the other way around: you're not getting `analytics` in the `render` function parameters, but in the `constructor`. You'll need to swap this.